community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -134,6 +134,10 @@
                 "<true/false>",
                 "Whether or not fields from input source can span multiple lines, i.e. contain newline characters." ),
 
+        TRIM_STRINGS( "trim-strings", org.neo4j.csv.reader.Configuration.DEFAULT.trimStrings(),
+                "<true/false>",
+                "Whether or not fields with id-type string should be trimmed for whitespaces."),
```

Comment:
This description is slightly misleading. It mentions that it trims fields with id-type string, but it seems that all fields of type string/string array are trimmed. `id-type` is a term for the `:ID`, `:START_ID`, `:END_ID` fields. It's probably enough saying something like:

`"Whether or not strings should be trimmed for whitespaces."